docs/src/components/ComponentDoc/ComponentControls/ComponentControlsCodeSandbox/createPackageJson.ts

```diff
@@ -3,20 +3,23 @@ import * as _ from 'lodash'
 import { imports } from 'docs/src/components/Playground/renderConfig'
 import { ComponentSourceManagerLanguage } from 'docs/src/components/ComponentDoc/ComponentSourceManager'
 
+const packageJson = require('../../../../../../package.json')
+const codeSandboxPackageJson = require('../../../../../../packages/code-sandbox/package.json')
+
 const name = 'stardust-ui-example'
 const description =
   'An exported example from Stardust UI React, https://stardust-ui.github.io/react/'
 const dependencies = {
-  '@stardust-ui/code-sandbox': 'latest',
-  ..._.mapValues(imports, () => 'latest'),
+  '@stardust-ui/code-sandbox': codeSandboxPackageJson.version,
+  ..._.mapValues(imports, (value, key, object) => value.version),
```

Comment:
```suggestion
  ..._.mapValues(imports, (package) => package.version),
```

Reply:
`package` is a reserved word in strict mode FYI